modules/http4s-munit/src/main/scala/munit/Http4sSuite.scala

```diff
@@ -27,15 +27,25 @@ abstract class Http4sSuite[A: Show] extends CatsEffectSuite {
    * @param testOptions the options for the current test
    * @return the test's name
    */
-  def munitHttp4sNameCreator(request: ContextRequest[IO, A], testOptions: TestOptions): String = {
+  def munitHttp4sNameCreator(
+      request: ContextRequest[IO, A],
+      testOptions: TestOptions,
+      repetitions: Option[Int],
+      maxParallel: Option[Int]
```

Comment:
Why not hand them as `config: Config`?

Reply:
See [`9516839` (#42)](https://github.com/alejandrohdezma/http4s-munit/pull/42/commits/9516839b75d13e9700891f6c0308038d71ab3c81)